gui/slick/interfaces/default/inc_bottom.mako

```diff
@@ -62,6 +63,14 @@
                 &nbsp;/&nbsp;<span class="footerhighlight">${ep_total}</span> Episodes Downloaded ${ep_percentage}
         | Daily Search: <span class="footerhighlight">${str(sickbeard.dailySearchScheduler.timeLeft()).split('.')[0]}</span>
         | Backlog Search: <span class="footerhighlight">${str(sickbeard.backlogSearchScheduler.timeLeft()).split('.')[0]}</span>
+
+        % if sickbeard.DEVELOPER:
+        <div>
+            Memory used: <span class="footerhighlight">${sickbeard.helpers.pretty_filesize(resource.getrusage(resource.RUSAGE_SELF).ru_maxrss)}</span> |
+            Load time: <span class="footerhighlight">${"%.4f" % (time() - sbStartTime)}s</span> / Mako: <span class="footerhighlight">${"%.4f" % (time() - makoStartTime)}s</span> |
+            Branch: <span class="footerhighlight">${sickbeard.BRANCH}</span>
+        </div>
+        % endif
     </div>
```

Comment:
That was new, right? I think that's the stuff @MGaetan89 just added. If he's removing the inc_ files I'll let him add this. Other than that I don't think anyone else has touched any mako files since I did my big conversion so we should be good if we aren't I'm sure @fernandog will fine the broken shit like normal. :stuck_out_tongue_winking_eye: